weavetest/doc.go

```diff
@@ -0,0 +1,9 @@
+/*
+Package weavetest provides mocks and functions that makes testing weave
+functionality easier.
+
+Before adding a new code to this package make sure that it is used in at least
+two packages. Mocks and helpers that are utilized by only one package should be
```

Comment:
Ahhh... this explains some of the questions before about mocks going there.

Great to add these doc strings

Reply:
I belong to the [`WET`](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#DRY_vs_WET_solutions) church! :church: :sweat_drops: :smiling_imp: